src/set.cr

```diff
@@ -203,6 +203,17 @@ struct Set(T)
     set
   end
 
+  # Addition: returns a new, unique set containing the elements from both sets.
+  # Because sets cannot have duplicates, this is equivalent to the union of
+  # both sets.
+  #
+  # ```
+  # Set{1, 1, 2, 3} + Set{3, 4, 5} # => Set{1, 2, 3, 4, 5}
+  # ```
```

Comment:
I do think the documentation could follow the methods `add` and `inspect` and just mention as ```# Alias for `|` ```. 

@RX14 about the docs, the methods `add` and `inspect` are aliases for `<<` and `to_s` respectively. 

Not making clear that `def +(other)` is an alias for the pipe(`Set#|`) might be confusing with `Set#add`, even that implementation and conceptually they are different.

Reply:
I agree that we should leave the alias note, but am willing to do either -- thoughts @RX14 ?

Reply:
This is not an alias, just two operators which happen to have the same semantics on this one datatype.